workflows/integration-testing/templates/imaging-workflow.yaml

```diff
@@ -47,8 +47,29 @@ spec:
             value: "-A Test_Report_MainTel_Housekeeping.list"
           - name: jobname
             value: maintel-housekeeping
-      - name: auxtel-image-verification
```

Comment:
Image verification needs to go before the daytime checkout. In principle, they are now doing the same thing.